### DISCUSSION TITLE
Urgent Request for Community Assistance - Hacked GitHub Account and Flagged

### DISCUSSION THREAD

**Discussion by CodeLife234** [community/community#109644] [New to GitHub] [answered]
_**NOTE: I am raising this discussion on my friend's flagged GitHub account.**_

I hope this message finds you well. On Wednesday, I faced a distressing situation involving my GitHub account (https://github.com/WayneMusungu), which was hacked and subsequently flagged. During this unauthorized access, two repositories were created and pushed to my account under the names "backup29d8b7f9-0562-45c3-a3d8-29e835a0a467" and "backupc061fd12-0a4d-480b-b3a0-41994af140b2." To make matters worse, a message was left, stating: "I hope this message finds you well. This is an urgent notice to inform you that your data has been compromised, and we have secured a backup with this link https://telegra.ph/URGENT-NOTICE-02-01." Additionally, my account and projects are no longer publicly visible, and some repositories have been deleted.

The incident occurred after clicking on a spam recruiting email, leading to the compromise of my account.

I am reaching out to the community for prompt assistance in resolving this matter. Your support and guidance would be highly appreciated. I raised the issue and here is the ticket number [Ticket 2615299](https://support.github.com/ticket/2615299).

@queenofcorgis , @davecheney, and @canuckjacq could you assist me with this?

Regards, 
Wayne

**Comment by Mindgames** [comment]
Exactly the same here

This is the email in question:
<img width="1421" alt="image" src="https://github.com/community/community/assets/2171273/65547a50-ab1f-4116-afef-c551c0dde9e8">

Trying to locate the Application / OAuth the was added.

**Reply by Mindgames** [reply]
<img width="962" alt="image" src="https://github.com/community/community/assets/2171273/e6eb9542-2f15-4a73-af72-d126d5dfd055">

**Reply by CodeLife234** [reply]
I received a similar email

**Reply by CodeLife234** [reply]
He has already revoked access for that.

**Comment by CodeLife234** [comment]
> _**NOTE: I am raising this discussion on my friend's flagged GitHub account.**_
> 
> I hope this message finds you well. On Wednesday, I faced a distressing situation involving my GitHub account (https://github.com/WayneMusungu), which was hacked and subsequently flagged. During this unauthorized access, two repositories were created and pushed to my account under the names "backup29d8b7f9-0562-45c3-a3d8-29e835a0a467" and "backupc061fd12-0a4d-480b-b3a0-41994af140b2." To make matters worse, a message was left, stating: "I hope this message finds you well. This is an urgent notice to inform you that your data has been compromised, and we have secured a backup with this link https://telegra.ph/URGENT-NOTICE-02-01." Additionally, my account and projects are no longer publicly visible, and some repositories have been deleted.
> 
> The incident occurred after clicking on a spam recruiting email, leading to the compromise of my account.
> 
> I am reaching out to the community for prompt assistance in resolving this matter. Your support and guidance would be highly appreciated. I raised the issue and here is the ticket number [Ticket 2615299](https://support.github.com/ticket/2615299).
> 
> @queenofcorgis , @davecheney, and @canuckjacq could you assist me with this?
> 
> Regards, Wayne



I had initially commented that I can't access the forms for application
![MicrosoftTeams-image](https://github.com/community/community/assets/153800509/c69f2dfe-fd17-4b95-a424-768ec21c8a21)


Then later I saw my account hacked and two repositories created.

![MicrosoftTeams-image (1)](https://github.com/community/community/assets/153800509/f277710b-4df0-44b4-a3aa-2c6741230fc3)

**Reply by suhdev** [reply]
I've had a similar attack on my account

**Comment by billythekid** [comment]
Hi @KhwiseroDev Please ask our friend to open a support ticket for reinstatement and obviously please do not click any links they've been sent.

Please also read over the accepted answer on this post: https://github.com/orgs/community/discussions/109171

**Reply by CodeLife234** [reply]
Here is the ticket that he created yesterday

![MicrosoftTeams-image (2)](https://github.com/community/community/assets/153800509/eda2d233-14e9-4209-a26d-85d440084b22)

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/orgs/community/discussions/109171

**Comment by ettaboyle** [comment]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Reply by VirtualCard** [reply]
Last morning, everything was going well. All repositories were working well, and I had updated some of them. After a few hours, when I came back, my account was logged out, and when I tried to log in, I was notified that my username or password was incorrect. I have recovered my password, and then I checked that my account is flagged. What can I do now? I need to get all my repositories back soon; it's really important to me. Ticket:2652333

**Comment by Buddhist-Ham** [comment]
Hello Guys!! 😁

ND here. It seems the Writers of the biblical events is still doing lots of last minute amendments. Would it be possible for you guys to help me update whoever is doing this rush job to hasten their resolve given that I had been on YouTube for the past 6 years slowly delivering answers to them.. 

![IMG_20220615_185725](https://github.com/community/community/assets/125282283/3045a48f-4013-44b6-bfb0-cb1292b21f27)
![Screenshot_2023_0925_135541](https://github.com/community/community/assets/125282283/1950faed-7c8e-4154-a505-44ecd5720f35)
![IMG_20221102_100040](https://github.com/community/community/assets/125282283/e81cdf3f-a4ee-488f-99cc-5f430912318a)
![Screenshot_2023_0925_135318](https://github.com/community/community/assets/125282283/c8cbba93-cce6-4a52-9db4-e57bfbde7d0c)
![IMG_20221102_100212~2](https://github.com/community/community/assets/125282283/d65ba3c1-60dd-48bb-9a13-61188a3d741c)
![IMG_20221102_100111~2](https://github.com/community/community/assets/125282283/9363133e-fbac-44a9-a2f8-f9c6589865af)

![IMG_20220615_185757](https://github.com/community/community/assets/125282283/8f5addf7-13d1-4cf7-aed5-5569d1493ef0)
![IMG_20220615_185837](https://github.com/community/community/assets/125282283/45c4b32d-5f14-43c5-8a62-83580a2c5830)
![IMG_20231204_185416](https://github.com/community/community/assets/125282283/304f30f3-13e6-48d7-955a-9bd8626fef5f)
![IMG_20221130_213452](https://github.com/community/community/assets/125282283/a0f0d638-c434-44b5-982f-95ee616fdd85)
![IMG_20230418_150105](https://github.com/community/community/assets/125282283/074f10fe-61ae-4b03-b168-2887ae052933)
![IMG_20221102_092729](https://github.com/community/community/assets/125282283/de2f1138-ea02-42bb-ae19-05af8e1adfa5)
![IMG_20221206_165947](https://github.com/community/community/assets/125282283/e1a300a3-1781-410b-b56c-3e4747614a50)
![IMG_20221102_100117~2](https://github.com/community/community/assets/125282283/a9a44e58-ec23-4831-9e44-54caf67cae3d)

**Comment by Buddhist-Ham** [comment]
I will leave my contact with you guys and gals as well just in case there is a necessity to get in touch with me.
I am in Malaysia btw +6016-339 6963, ANDY

Best Regards 😘
杨耀君 (Yong Yew Kuan) a.k.a. 😭 Christ the Lord...

**Comment by Aboodkaaboodkaaaa** [comment]
L

احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: Buddhist-Ham ***@***.***>
‏‏تم الإرسال: Monday, March 4, 2024 12:27:52 PM
إلى: community/community ***@***.***>
نسخة: Subscribed ***@***.***>
‏‏الموضوع: Re: [community/community] Urgent Request for Community Assistance - Hacked GitHub Account and Flagged (Discussion #109644)


I will leave my contact with you guys and gals as well just in case there is a necessity to get in touch with me.
I am in Malaysia btw +6016-339 6963, ANDY

Best Regards 😘
杨耀君 (Yong Yew Kuan) a.k.a. 😭 Christ the Lord...

**Comment by Aboodkaaboodkaaaa** [comment]
احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: Buddhist-Ham ***@***.***>
‏‏تم الإرسال: Monday, March 4, 2024 12:27:52 PM
إلى: community/community ***@***.***>
نسخة: Subscribed ***@***.***>
‏‏الموضوع: Re: [community/community] Urgent Request for Community Assistance - Hacked GitHub Account and Flagged (Discussion #109644)


I will leave my contact with you guys and gals as well just in case there is a necessity to get in touch with me.
I am in Malaysia btw +6016-339 6963, ANDY

Best Regards 😘
杨耀君 (Yong Yew Kuan) a.k.a. 😭 Christ the Lord...

**Comment by Aboodkaaboodkaaaa** [comment]
احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: Buddhist-Ham ***@***.***>
‏‏تم الإرسال: Monday, March 4, 2024 12:22:17 PM
إلى: community/community ***@***.***>
نسخة: Subscribed ***@***.***>
‏‏الموضوع: Re: [community/community] Urgent Request for Community Assistance - Hacked GitHub Account and Flagged (Discussion #109644)


Hello Guys!! 😁

ND here. It seems the Writers of the biblical events is still doing lots of last minute amendments. Would it be possible for you guys to help me update whoever is doing this rush job to hasten their resolve given that I had been on YouTube for the past 6 years slowly delivering answers to them..

IMG_20220615_185725.jpg (view on web)<https://github.com/community/community/assets/125282283/3045a48f-4013-44b6-bfb0-cb1292b21f27>
Screenshot_2023_0925_135541.jpg (view on web)<https://github.com/community/community/assets/125282283/1950faed-7c8e-4154-a505-44ecd5720f35>
IMG_20221102_100040.jpg (view on web)<https://github.com/community/community/assets/125282283/e81cdf3f-a4ee-488f-99cc-5f430912318a>
Screenshot_2023_0925_135318.jpg (view on web)<https://github.com/community/community/assets/125282283/c8cbba93-cce6-4a52-9db4-e57bfbde7d0c>
IMG_20221102_100212.2.jpg (view on web)<https://github.com/community/community/assets/125282283/d65ba3c1-60dd-48bb-9a13-61188a3d741c>
IMG_20221102_100111.2.jpg (view on web)<https://github.com/community/community/assets/125282283/9363133e-fbac-44a9-a2f8-f9c6589865af>

IMG_20220615_185757.jpg (view on web)<https://github.com/community/community/assets/125282283/8f5addf7-13d1-4cf7-aed5-5569d1493ef0>
IMG_20220615_185837.jpg (view on web)<https://github.com/community/community/assets/125282283/45c4b32d-5f14-43c5-8a62-83580a2c5830>
IMG_20231204_185416.jpg (view on web)<https://github.com/community/community/assets/125282283/304f30f3-13e6-48d7-955a-9bd8626fef5f>
IMG_20221130_213452.jpg (view on web)<https://github.com/community/community/assets/125282283/a0f0d638-c434-44b5-982f-95ee616fdd85>
IMG_20230418_150105.jpg (view on web)<https://github.com/community/community/assets/125282283/074f10fe-61ae-4b03-b168-2887ae052933>
IMG_20221102_092729.jpg (view on web)<https://github.com/community/community/assets/125282283/de2f1138-ea02-42bb-ae19-05af8e1adfa5>
IMG_20221206_165947.jpg (view on web)<https://github.com/community/community/assets/125282283/e1a300a3-1781-410b-b56c-3e4747614a50>
IMG_20221102_100117.2.jpg (view on web)<https://github.com/community/community/assets/125282283/a9a44e58-ec23-4831-9e44-54caf67cae3d>
